spec/models/journaled/client_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe Journaled::Client do
+  describe ".generate" do
```

Comment:
doth quoth the ~~raven~~ :owl:

Reply:
ah, I see, I think a lot of these double quotes came from the older, now deleted specs